File: code/1020/1020.cpp
#include <iostream>
using namespace std;

int main() {

  int n, years, months, days;

  cin >> n;

  years = n / 365;

  months = (n - (years * 365)) / 30;

  days = n - (years * 365) - (months * 30);

  cout << years << " ano(s)" << endl
       << months << " mes(es)" << endl
       << days << " dia(s)" << endl;

  return 0;
}

File: code/1020/1020.rb
n = gets.to_i

years = (n / 365).to_i
months = ((n - (years * 365)) / 30).to_i
days = (n - (years * 365) - (months * 30)).to_i

puts format('%d ano(s)', years)
puts format('%d mes(es)', months)
puts format('%d dia(s)', days)
